src/style.css

```
@@ -0,0 +1,88 @@
+body {
+  margin: 0;
+}
+
+.flex-container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  line-height: 300px;
+  min-width: 300px;
+  text-align: center;
+  font-size: 100px;
+  color: white;
+  font-weight: bold;
+  font-family: Arial, Arial, Helvetica, sans-serif;
```

Don't repeat the same families.
```suggestion
  font-family: Arial, Helvetica, sans-serif;
```